src/Command/ConsumeCommand.php

```diff
@@ -38,7 +38,7 @@ public function configure()
             ->addOption('max-runtime', null, InputOption::VALUE_OPTIONAL, 'Maximum time in seconds the consumer will run.', null)
             ->addOption('max-messages', null, InputOption::VALUE_OPTIONAL, 'Maximum number of messages that should be consumed.', null)
             ->addOption('stop-when-empty', null, InputOption::VALUE_NONE, 'Stop consumer when queue is empty.', null)
-            ->addArgument('queue', InputArgument::REQUIRED, 'Name of queue that will be consumed.')
+            ->addArgument('queue', InputArgument::REQUIRED, 'Names of one or more queues that will be consumed, separated by commas.')
```

Comment:
Maybe use this instead? https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Console/Input/InputArgument.php#L26